### ISSUE TITLE
Disable Pass Robot arguments using argument file throw error 

### ISSUE THREAD

**Issue by kenifanying** [nokia/RED#284]
Hi, 

My PYTHONPATH contain some Chinese character.  If I run test, Red console will throw an error after preference: 'Robot Framework -> Launching->Pass Robot arguments using argument file' is disabled, see some output below:
```
Command: C:\Python27\python.exe -m robot.run --listener C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py:49902 -P C:\Python27\Lib\site-packages:C:\Users\tony\robotframework\rf_pi:C:\Users\tony\robotframework\rf_pi\switch\子系统:C:\Python27\Lib -s "Rf Pi.Pi Eg.Content Audit.Https.TC JR CJ HTTPS 01" -t "Rf Pi.Pi Eg.Content Audit.Https.TC JR CJ HTTPS 01.tc_jr_cj_https_01_01" --variable cfg_name:cfg_test1 C:\Users\tony\robotframework\rf_pi
Suite Executor: Robot Framework 3.1.1 (Python 2.7.16 on win32)
Traceback (most recent call last):
  File "C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py", line 683, in _send_to_server
    self.decoder_encoder.dump(packet)
  File "C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py", line 741, in dump
    self._write(self._json_encoder(obj))
  File "C:\Python27\Lib\json\encoder.py", line 209, in encode
    chunks = list(chunks)
  File "C:\Python27\Lib\json\encoder.py", line 434, in _iterencode
    for chunk in _iterencode_dict(o, _current_indent_level):
  File "C:\Python27\Lib\json\encoder.py", line 408, in _iterencode_dict
    for chunk in chunks:
  File "C:\Python27\Lib\json\encoder.py", line 332, in _iterencode_list
    for chunk in chunks:
  File "C:\Python27\Lib\json\encoder.py", line 390, in _iterencode_dict
    yield _encoder(value)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xd7 in position 993: invalid continuation byte
[ ERROR ] Taking listener 'C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py:49902' into use failed: Importing listener 'C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py' failed: Creating instance failed: UnicodeDecodeError: 'utf8' codec can't decode byte 0xd7 in position 993: invalid continuation byte
Traceback (most recent call last):
  File "C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py", line 235, in __init__
    self._handshake()
  File "C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py", line 244, in _handshake
    self._send_version()
  File "C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py", line 269, in _send_version
    self._send_to_server(AgentEventMessage.VERSION, info)
  File "C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py", line 683, in _send_to_server
    self.decoder_encoder.dump(packet)
  File "C:\Users\tony\AppData\Local\Temp\RobotTempDir1134876472379339834\TestRunnerAgent.py", line 741, in dump
    self._write(self._json_encoder(obj))
  File "C:\Python27\Lib\json\encoder.py", line 209, in encode
    chunks = list(chunks)
  File "C:\Python27\Lib\json\encoder.py", line 434, in _iterencode
    for chunk in _iterencode_dict(o, _current_indent_level):
  File "C:\Python27\Lib\json\encoder.py", line 408, in _iterencode_dict
    for chunk in chunks:
  File "C:\Python27\Lib\json\encoder.py", line 332, in _iterencode_list
    for chunk in chunks:
  File "C:\Python27\Lib\json\encoder.py", line 390, in _iterencode_dict
    yield _encoder(value)
```

It's ok if I enable this option, any help would be appreciated.

Python 2.7.16
Robot Framework 3.1.1 (Python 2.7.16 on win32)
RED - Robot Editor version 0.8.11
System: Windows 7 SP1 X86_64

**Comment by michalanglart** [comment]
Hi,

yes this looks like a bug on our side. We'll see what to do to fix that.

**Comment by michalanglart** [comment]
Fixed in 0.8.12
